Makefile

```diff
@@ -178,6 +182,20 @@ bundle-build:
 .PHONY: deploy_bundle
 deploy_bundle: undeploy_bundle
 	operator-sdk run bundle $(IMAGE_NAMESPACE)/$(OPERATOR_NAME)-bundle:$(IMAGE_VERSION)
+ifeq ($(DISABLE_SERVICE_TLS), true)
+	@echo "Disabling TLS for in-cluster communication between Services"
+	@current_ns=`$(CLUSTER_CLIENT) config view --minify -o 'jsonpath={.contexts[0].context.namespace}'` && \
```

Comment:
This seems like a good and more generic way to check for the current namespace rather than using `oc project -q`, which I had documented in the `README.md` before. Maybe this can be document there instead instead/as well?

Reply:
Sure, I changed it in the README.